pmm-app/src/pmm-qan/panel/provider/provider.tools.ts

```diff
@@ -128,9 +128,7 @@ export const refreshGrafanaVariables = (state) => {
     variablesQuery.search = state.search;
   }
 
-  getLocationSrv().update({
-    query: variablesQuery,
-  });
+  locationService.partial(variablesQuery);
```

Comment:
See examples [here](https://grafana.com/developers/plugin-tools/migration-guides/update-from-grafana-versions/migrate-8_3_x-to-8_4_x)